tests/CodeSnifferRunner.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace ObjectCalisthenics\Tests;
+
+use PHP_CodeSniffer;
+
+final class CodeSnifferRunner
+{
+    /**
+     * @var PHP_CodeSniffer
+     */
+    private $codeSniffer;
+
+    public function __construct()
+    {
+        $this->codeSniffer = new PHP_CodeSniffer();
+    }
+
+    /**
+     * @param string $testedFile
+     * @param string $sniffName
+     *
+     * @return int
+     */
+    public function detectErrorCountInFileForSniff($testedFile, $sniffName)
+    {
+        $this->codeSniffer->initStandard('../../../../../src/ObjectCalisthenics', [$sniffName]);
```

Comment:
Ops... missed the notification, thanks for pinging me

Which `ruleset.xml`? Because for the one contained in this repo I suggest to use just the `__DIR__`, instead for all the directories there's the attribute [here](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer.php#L92-L102) (even if I don't know how to access to it) or again the `get_included_files()`.